src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -9,10 +13,6 @@
 using System.IO.Abstractions;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.Azure.Templates.Analyzer.Core;
-using Microsoft.Azure.Templates.Analyzer.Types;
-using Microsoft.Azure.Templates.Analyzer.Reports;
-using Newtonsoft.Json.Linq;
```

Comment:
Please revert the change in ordering.

Reply:
> System.* namespaces should be listed first

See https://github.com/Azure/template-analyzer/blob/development/CONTRIBUTING.md#coding-conventions